cmake/projects/NASM/schemes/url_sha1_nasm_unpack_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
> No, it's unpacked into prefix/bin instead of prefix

I don't see anything difference between new `url_sha1_nasm_unpack_install_windows` and [url_sha1_unpack](https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_unpack.cmake.in).